src/oc-id/app/controllers/profiles_controller.rb

```diff
@@ -22,9 +22,9 @@ def update
     # Instead send a confirmation email and let that email link update the
     # email address.
     #update note
-    params[:user].reject! { |k| k == :email }
+    params[:user].reject! { |k| k == "email" }
 
-    if @user.update_attributes(params[:user])
+    if @user.update_attributes(user_params)
```

Comment:
Do we need this change as a part of the rails upgrade?

Reply:
yes, we need to do this for rails upgrade.